app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -84,6 +84,18 @@
     private Place place;
 
     private boolean isExpanded = true;
+    /**
```

Comment:
Tiny change request: One empty line between the javadoc and the previous declaration.

Reply:
Done!